### PR TITLE
xxhashport can now be built

### PR DESCRIPTION
**Pull request by MikeFultonDev**

### CODE REVIEW

File: bin/zopen-build [hunk 1]
```diff
@@ -464,6 +466,8 @@ extractTarBall()
   if ! git init . >/dev/null || ! git add -f ${files} >/dev/null || ! git commit --allow-empty -m "Create Repository for patch management" >/dev/null; then
     printError "Unable to initialize git repository for tarball"
   fi
+
```
Comment (IgorTodorovskiIBM) [inline bin/zopen-build:469]: Minor: extra spaces?

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

File: bin/zopen-build (hunk 1 above)
Comment (MikeFultonDev) [reply bin/zopen-build:469]: i am not seeing them... where do you see the extra spaces?

Comment (IgorTodorovskiIBM) [reply bin/zopen-build:469]: I mean extra newlines (line 469 and 470)?

**Review by ccw-1 — APPROVED** [review]

**Pull request merged**